decidim-proposals/lib/decidim/proposals/seeds.rb

```diff
@@ -60,6 +63,7 @@ def create_component!
           settings: {
             vote_limit: 0,
             attachments_allowed: [true, false].sample,
+            amendments_enabled: [true, false].sample,
```

Comment:
Adding the `[true, false].sample` can create situations where no component has the amendments enabled or disabled. In my case, in my seed data while testing this, i got only 1 component with amendments disabled. 

```suggestion
            amendments_enabled: participatory_space.id.odd?,
```